coriolis/providers/replicator.py

```diff
@@ -34,13 +32,20 @@
 
 DEFAULT_REPLICATOR_PORT = 4433
 
+request_opts = [
+    cfg.IntOpt('default_requests_timeout',
```

Comment:
Imo it would still be useful to have a separate `CONF.replicator. default_requests_timeout` option for granularity's sake.